test/specs/entityChevron.js

```diff
@@ -0,0 +1,117 @@
+        for (let i = 0; i < $$(selectors.common.recordActualData).length; i++) {
+            expect($$(selectors.common.recordActualData)[i].getText())
+                .to.equal(expected.chevron.expectedData[i]);
+        };
+    });
+    it('08/CE Should activate view mode and verify that record is displayed with right data', () => {
+        clickActionView();
+        expect($(selectors.chevron.recordCardBody).isDisplayed()).true;
+        // for (let i = 0; i < expected.chevron.expectedData.length; i++) {
+        //     expect($(selectors.chevron.viewModeRecordCard).getText())
+        //         .to.equal(expected.chevron.expectedData[i]);
+        // };
+        expect($("//span[text()='Pending']").isExisting()).true;
+        expect($("//span[text()='Hello']").isExisting()).true;
+        expect($("//span[text()='10']").isExisting()).true;
+        expect($("//span[text()='1.10']").isExisting()).true;
+        expect($("//span[text()='15/06/2021']").isExisting()).true;
+        expect($("//span[text()='15/06/2021 10:10:10']").isExisting()).true;
+        expect($(selectors.chevron.viewModeUserName).isExisting()).true;
+        $(selectors.chevron.closeRecordBtn).click();
+     });
+    it('09/CE Should activate edit mode', () => {
+        clickActionEdit();
+        expect($(selectors.chevron.recordCardBody).isDisplayed()).true;
+    });
+    it('10/CE Should fill all fields with new data and save new record card', () => {
+        $(selectors.chevron.textArea).click();
+        $(selectors.chevron.textArea).setValue(testData.chevron.areaValueEdit[0]);
+        $(selectors.chevron.intArea).click();
+        $(selectors.chevron.intArea).setValue(testData.chevron.areaValueEdit[1]);
+        $(selectors.chevron.decimalArea).click();
+        $(selectors.chevron.decimalArea).setValue(testData.chevron.areaValueEdit[2]);
+        $(selectors.chevron.dateArea).click();
+        $(selectors.chevron.dateArea).clearValue();
+        $(selectors.chevron.dateArea).setValue(testData.chevron.areaValueEdit[3]);
+        $(selectors.chevron.timeArea).click();
+        $(selectors.chevron.timeArea).clearValue();
+        $(selectors.chevron.timeArea).setValue(testData.chevron.areaValueEdit[4]);
+        $(selectors.common.saveBtn).click();
+        expect(getRowCount()).to.equal(expected.chevron.expectedRecordRow[1]);
+        expect($(selectors.chevron.footerTableOfRecords).getText())
+            .to.equal(expected.chevron.footerOfOneRecordTable);
+    });
+    it('11/CE Should verify that edited record was saved with right data', () => {
+        for (let i = 0; i < $$(selectors.common.recordActualData).length; i++) {
+            expect($$(selectors.common.recordActualData)[i].getText())
+                .to.equal(expected.chevron.expectedEditedData[i]);
+        };
+    });
+    it('12/CE Should delete record', () => {
+        clickActionDelete();
+        expect(getRowCount()).to.equal(expected.chevron.expectedRecordRow[0]);
```

Comment:
```suggestion
        expect(getRowCount()).to.equal(expected.common.noRecord);
```
check all **common** in expected, selectors, testData